crates/resolver/src/config.rs

```diff
@@ -785,6 +785,8 @@ pub struct ResolverOpts {
     pub preserve_intermediates: bool,
     /// Try queries over TCP if they fail over UDP.
     pub try_tcp_on_error: bool,
+    /// Shuffle DNS servers before each query.
```

Comment:
Minor nit, can you make it clear that this is a random ordering, i.e. not doing round-robin or anything?

Reply:
```suggestion
    /// Shuffle DNS servers before each query. It uses a random shuffle algorithm from the `rayon` crate.
```

Reply:
Do you think that it's ok?